src/panels/config/automation/dialog-new-automation.ts

```diff
@@ -33,9 +34,13 @@ class DialogNewAutomation extends LitElement {
     this._opened = true;
   }
 
-  public closeDialog(): void {
+  public async closeDialog(): Promise<void> {
```

Comment:
@bramkragten is this a standardized method?

Reply:
Yes

Reply:
Use `implements HassDialog` and you'll get an error now :-)